src/main/webapp/dev/js/common/instructorEnroll.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * Retrieves updated column header order and generates a header string.
+ *
+ * Example: Changes this array ['Section', 'Team', 'Name', 'Email', 'Comments']
+ * into a string = "Section|Team|Name|Email|Comments\n"
+ *
+ * @returns {string} updated header string
+ */
+function getUpdatedHeaderString(handsontableColHeader) {
+    const colHeaders = handsontableColHeader.join('|');
+    return colHeaders.concat('\n');
+}
+
+/**
+ * Retrieves user data rows rows in the spreadsheet interface and transforms it into a string.
+ *
+ * Null value from cell is changed to empty string after .join(). Filters empty rows in the process.
+ *
+ * Example:
+ * 2 by 5 spreadsheetData (before)
+ * ['TestSection1', 'Team1', 'null', 'test1@xample.com', 'test1comments']
+ * ['TestSection2', null, 'TestName2', 'test2@example.com', null]
+ *
+ * 2 by 5 spreadsheetData (after)
+ * "TestSection1|Team1||test1@xample.com|test1comments\n
+ *  TestSection2||TestName2|test2@example.com|\n"
+ *
+ * @returns {string} user data rows
+ */
+function getUserDataRows(spreadsheetData) {
+    // needs to check for '' as an initial empty row with null values will be converted to e.g. "||||" after .map
+    return spreadsheetData.filter(row => (!row.every(cell => cell === null || cell === '')))
+            .map(row => row.join('|'))
+            .join('\n');
+}
+
+/**
+ * Pushes data from spreadsheetDataRows into an array.
+ * Facilitates the function loadData for the Handsontable instance.
+ * @returns {Array} updated data
+ */
+function getUpdatedData(spreadsheetDataRows) {
+    return spreadsheetDataRows.map(row => row.split('|'));
+}
+
+/**
+ * Transforms the first uppercase letter of a string into a lowercase letter.
+ * @returns {string} string in all lowercase letters
+ */
+function unCapitalizeFirstLetter(string) {
+    return string.charAt(0).toLowerCase() + string.slice(1);
+}
+
+/**
+ * Converts returned AJAX data to a suitable format required by Handsontable.
+ * @returns {Array} student data
+ */
+function ajaxDataToHandsontableData(studentsData, handsontableColHeader) {
+    const headers = handsontableColHeader.map(unCapitalizeFirstLetter);
+    return studentsData.map(student => (headers.map(
+            header => student[header])));
+}
+
+/**
+ * Toggle the chevron image depending on the user's action.
+ */
+function toggleChevronImage(panelCollapse, toggleChevron) {
+    if ($(panelCollapse[0]).hasClass('checked')) { // panel is shown
```

Comment:
Looks like everytime you pass `panelCollapse`, you'll use `$(panelCollapse[0])`. If I'm not wrong, you shouldn't need to. Just use `panelCollapse` directly.